### PR TITLE
tests

### LINKED ISSUE #31: Tests
Let's start with a few known output tests for select municipalities, and also error behavior.

### PR DESCRIPTION
**Pull request by TSchiefer**
closes #31

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -47,6 +47,7 @@ Imports:
 Suggests:
     daff,
     knitr,
+    pkgload,
```
Comment (krlmlr) [inline DESCRIPTION:50]: What do we need this for?

File: tests/testthat/test-internal-consistency.R [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+csv_dir <- system.file("csv/flat", package = "munch")
+all_files <- list.files(csv_dir)
+file_paths <- file.path(csv_dir, all_files)
+
+all_data <-
+  file_paths %>%
+  purrr::map_df(~ read_mun_csv(.)) %>%
+  as_tibble() %>%
+  mutate(target_year = as.integer(target_year))
```
Comment (krlmlr) [inline tests/testthat/test-internal-consistency.R:19]: Should this be a function in `R/` instead?

File: tests/testthat/test-up-to-date.R [hunk 3]
```diff
@@ -1,26 +1,34 @@
...
+#   cantons <- data$canton
+#   district_mutations <- data$district
+#   municipality_mutations <- data$municipality
+#
+#   old <- usethis::proj_set(pkg_path)
+#
+#   usethis::use_data(
+#     cantons, district_mutations, municipality_mutations,
+#     overwrite = TRUE
+#   )
```
Comment (krlmlr) [inline tests/testthat/test-up-to-date.R:33]:
```suggestion
```

**Review by krlmlr — APPROVED** [review]
Thanks, looks good!

File: tests/testthat/test-internal-consistency.R (hunk 2 above)
Comment (TSchiefer) [reply tests/testthat/test-internal-consistency.R]: now an internal function: 8a2bb93

**Comment by TSchiefer** [comment]
think the Windows tests should now work with 4edd1bb -- currently failing due to https://chocolatey.org/api/v2/ being down for maintenance, checking later again.

File: DESCRIPTION (hunk 1 above)
Comment (krlmlr) [reply DESCRIPTION:50]: The example file shouldn't use it either.

**Comment by krlmlr** [comment]
It's tricky.

PRs are checked using a synthetic commit where the source branch is merged into the target branch: https://github.com/cynkra/munch/pull/44/checks?check_run_id=2237415606#step:3:394. The main branch was ahead and contained the evil pkgload line.

I think it's fine to use pkgload in the `data-raw/` directory.

**Comment by krlmlr** [comment]
To recover, `git reset --hard f809e25`, force-push, and redo.

**Comment by krlmlr** [comment]
We really need to merge main into this branch to avoid losing our minds.

**Comment by TSchiefer** [comment]
Sorry for the mess... when I locally merge master into this branch, no changes are the result. So I will let these checks finish, cause I haven't lost all hope that this will work now...

**Comment by krlmlr** [comment]
The main branch is called main.

**Comment by krlmlr** [comment]
...in this repo.

**Comment by TSchiefer** [comment]
you're right, I meant main; seems to be good now.

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**